app/src/main/java/com/pvp/app/ui/screen/goals/GoalScreen.kt

```diff
@@ -330,7 +330,7 @@ fun ProgressBar(goal: Goal) {
         Text(
             text = when (goal.steps) {
                 true -> "${goal.progress.toInt()} / ${goal.target.toInt()} steps"
-                false -> "${goal.progress} / ${goal.target} km"
+                false -> "${String.format("%.2f", goal.progress)} / ${goal.target} km"
```

Comment:
A shorter version, I believe, would be `"%.2f".format(goal.progress)`. Not required, just noting, maybe you didn't know and would prefer the shorter version, either is fine.

Reply:
Didn't know, thanks.